lib/barbeque/message/job_execution.rb

```diff
@@ -13,7 +13,11 @@ def assign_body(message_body)
         super
         @application = message_body['Application']
         @job  = message_body['Job']
-        @body = message_body['Message']
+        @body = if message_body['Message'].is_a?(String)
```

Comment:
Honestly there're no use case for us. So I should get rid of this fix for JobExecution message...